Tests/BrowserServicesKitTests/SecureVault/AutofillKeyStoreProviderTests.swift

```diff
@@ -0,0 +1,145 @@
+//
+//  AutofillKeyStoreProviderTests.swift
+//
+//  Copyright © 2024 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import XCTest
+import SecureStorage
+import SecureStorageTestsUtils
+@testable import BrowserServicesKit
+
+final class AutofillKeyStoreProviderTests: XCTestCase {
+
+    private enum Constants {
+        static let v1ServiceName = "DuckDuckGo Secure Vault"
+        static let v2ServiceName = "DuckDuckGo Secure Vault v2"
+        static let v3ServiceName = "DuckDuckGo Secure Vault v3"
+    }
+
+    private enum EntryName: String, CaseIterable {
+
+        case generatedPassword = "32A8C8DF-04AF-4C9D-A4C7-83096737A9C0"
+        case l1Key = "79963A16-4E3A-464C-B01A-9774B3F695F1"
+        case l2Key = "A5711F4D-7AA5-4F0C-9E4F-BE553F1EA299"
+
+        var keyValue: String {
+            (Bundle.main.bundleIdentifier ?? "com.duckduckgo") + rawValue
+        }
+    }
```

Comment:
You shouldn't be needing this, and just reuse types from AutofillKeyStoreProvider, right? (I'd say it's fine to mark `AutofillKeyStoreProvider.EntryName` internal vs private to expose it to the tests target).

Reply:
Done